diff --git a/modules/tracktion_graph/tracktion_graph/players/tracktion_NodePlayerUtilities.h b/modules/tracktion_graph/tracktion_graph/players/tracktion_NodePlayerUtilities.h
index c06b9161ce2..5dbad75e8ca 100644
--- a/modules/tracktion_graph/tracktion_graph/players/tracktion_NodePlayerUtilities.h
+++ b/modules/tracktion_graph/tracktion_graph/players/tracktion_NodePlayerUtilities.h
@@ -64,8 +64,8 @@ namespace node_player_utils
                 return idStrings;
             };
 
-            for (auto id : duplicateIDs)
-                DBG("\t" << id << ": " << getNodeTypeStrings (id));
+            // for (auto id : duplicateIDs)
+            //     DBG("\t" << id << ": " << getNodeTypeStrings (id));
         }
        #endif
 
